StabilityPackage.wl

```diff
@@ -54,6 +54,14 @@ getMoments::usage=
 "takes moments out of moment data for a given file, radius, species. Energy integrated"
 exportkadapt::usage=
 "[outevs_,name_]"
+ImportCalcGridData::usage=
+"Imports the calcultion grid elements."
+ImportCalcUniqueData::usage=

```

Comment:
I think we talked about this, but I don't think you want to store "unique" data, but rather just store all of the data.

Reply:
There's a reason I want this format (for analysis later) and the difference between exporting it in the unique format and importing it and deleting the repeated cases at the import step seems the same. Is there a reason it's better to do it on the import side rather than the export side if the other than file size (which isn't very big as it)?

Reply:
Yes - keeping the data format stupidly simple and totally complete. The storage space required isn't an issue at  ll, I think.

Reply:
Done!